index.html

```diff
@@ -19,10 +19,10 @@
         <h1> Ruby UY </h1>
         <h2> Comunidad de Ruby en Uruguay &#127482;&#127486; </h2>
         <ul>
-          <li>Meetup: <a href="https://www.meetup.com/ruby-montevideo/">Montevideo</a></li>
-          <li>Slack: <a href="https://owu.slack.com/archives/C52L2AK8W">Open Web Uruguay - Ruby</a></li>
-          <li>Sponsors: <a href="sponsors.html">List</a></li>
-          <li>Github: <a href="https://github.com/rubyuy/ruby.uy">Repo</a></li>
+          <li>Meetup: <a href="https://www.meetup.com/ruby-montevideo/" target="_blank">Montevideo</a></li>
+          <li>Slack: <a href="https://owu.slack.com/archives/C52L2AK8W" target="_blank">Open Web Uruguay - Ruby</a></li>
+          <li>Sponsors: <a href="sponsors.html" target="_blank">List</a></li>
```

Comment:
this one shouldn't open a new page since it's the same domain, right?

Reply:
Yeah, you're right, sorry!